azure-kusto-data/azure-kusto-data.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files are unrelated, and usually you won't commit them, but since they will help intellij automatically sort out the module issue for people it might worth it to actually commit them